gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -146,7 +148,7 @@ public final class InstantiatingGrpcChannelProvider implements TransportChannelP
   @VisibleForTesting final ImmutableMap<String, ?> directPathServiceConfig;
   @Nullable private final MtlsProvider mtlsProvider;
   @Nullable private final SecureSessionAgent s2aConfigProvider;
-  @Nullable private final List<HardBoundTokenTypes> allowedHardBoundTokenTypes;
```

Comment:
Can you set this in the constructor below? And Remove the Nullable tag?

Reply:
Done in https://github.com/googleapis/sdk-platform-java/pull/3591/commits/d8b2959f1e05d4158570a2d9af93ff061ac8f310